docs/sources/installation/helm.md

```diff
@@ -1,8 +1,11 @@
 ---
 title: Helm
+weight: 20
 ---
 # Install Loki with Helm
 
+The Helm installation runs the Loki cluster in microservices mode.
```

Comment:
The instructions here use helm charm which runs Loki in single binary mode. There is a `loki-distributed` chart that runs Loki in microservices mode but it is not covered here. Let us update the comment for now saying it runs Loki in single binary mode and then we can update it when we add the instructions for `loki-distributed`.

Reply:
Correction implemented.  Thanks.